clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -2716,6 +2721,18 @@ static void CollectArgsForIntegratedAssembler(Compilation &C,
           << "-Wa,--crel" << D.getTargetTriple();
     }
   }
+  if (Msa) {
+    if (!Triple.isMIPS()) {
+      if (*Msa)
+        D.Diag(diag::err_drv_unsupported_opt_for_target)
```

Comment:
`D.Diag(diag::err_drv_unsupported_opt_for_target)` << (*Msa ? "-Wa,-mssa" : "-Wa,-mno-ssa") << D.getTargetTriple()`

Reply:
OK, I applied it. Thanks!